result/flow.yaml

```diff
@@ -1,97 +1,116 @@
-openapi: 3.1.0
-
-info:
-  title: Flow results models
-  version: ^0.0.0
-
 components:
   schemas:
     Flow.Metrics.Request:
       description: >-
-        The request to the traffic generator for flow results.
+        The request container for flow metrics.
       type: object
       properties:
-        flow_names:
+        flow_groups:
```

Comment:
with `flow_groups` , user side complexity will increase. If I have 10 flows and want to retrieve 5 flows, have to create 5 groups.

Ex:
    request = api.metrics_request()
    f1, f2, f3, f4, f5 = request.flow.flowgroups.group().group().group().group().group()
    f1.flow_name = 'f1'
    f2.flow_name = 'f2'
    f3.flow_name = 'f3'
    f4.flow_name = 'f4'
    f5.flow_name = 'f5'
    return api.get_flow_metrics(request)

Reply:
aab67e5